### PR TITLE
Changes for Jitsi UI

### PR DESCRIPTION
**Pull request by nisarg1499**
Issue Solved : https://github.com/codeuino/social-platform-donut-frontend/issues/516
Image of UI after changes
jitsi2
Allignment is done and height issue is also solved. 

Please review : @jaskiratsingh2000 @vaibhavdaren

### CODE REVIEW

**Comment by jaskiratsingh2000** [comment]
LGTM!

**Comment by AuraOfDivinity** [comment]
@jaskiratsingh2000 @vaibhavdaren  A small concern from my end, Is this the suggested button format that we are planning to use within the other Modals used throughout the platform?

**Comment by devesh-verma** [comment]
> @jaskiratsingh2000 @vaibhavdaren A small concern from my end, Is this the suggested button format that we are planning to use within the other Modals used throughout the platform?

Nope. It should match with other button designs

**Comment by nisarg1499** [comment]
> @nisarg1499 Please use the css classes which is used in all the other modals.
> Just have a look at **modals.scss.**
> You don't need to write css for this. Also for this part why you modified **comments.scss** ??
> It would have effect the comment modals as well. Please look at it.

Actually the css of comment file was being used in join call button. But now I have updated the css of jitsi and now it's independent. And the design element is also same as others.
Output after changes : 
new-jitsi

File: src/user/integrations/NameForm.js [hunk 1]
```diff
@@ -55,14 +55,16 @@ export class Info extends Component{
...
-              disabled={this.state.disable} className="savebtn">Join Call</Button>
-            {this.state.startJitsi ? <JitsiMeet show={this.state.startJitsi}
-            onHide={this.closeJitsi} user={this.state.user} 
-            roomID={this.state.roomID}/> : null}
-            <Button variant="outline-primary" onClick={this.props.onHide}>Cancel</Button>
-          </div>
+          <Modal.Body>
+            <div className="form-footer-jitsi">
+              <Button onClick={() => this.setState({startJitsi:true})} 
+                disabled={this.state.disable} className="joinbtn">Join Call</Button>
```
Comment (Rupeshiya) [inline src/user/integrations/NameForm.js:61]: I think you are missing one class for the Button text. Please check once in other component

File: src/user/integrations/NameForm.js [hunk 2]
```diff
@@ -55,14 +55,16 @@ export class Info extends Component{
...
+              {this.state.startJitsi ? <JitsiMeet show={this.state.startJitsi}
+              onHide={this.closeJitsi} user={this.state.user} 
+              roomID={this.state.roomID}/> : null}
+              <Button variant="outline-primary" onClick={this.props.onHide}>Cancel</Button>
```
Comment (Rupeshiya) [inline src/user/integrations/NameForm.js:65]: Same here

File: src/user/integrations/NameForm.js (hunk 1 above)
Comment (nisarg1499) [reply src/user/integrations/NameForm.js:61]: Added the same styles in css of both buttons.

**Comment by nisarg1499** [comment]
Final UI.
finalll

File: src/user/integrations/jitsi.scss [hunk 3]
```diff
@@ -52,34 +52,37 @@
...
-    //margin-left: 3px;
+    margin-left: 3px;
 }
 .form-content{
     margin-top:-2%;
     margin-bottom: 20px;
     justify-content:space-between;
 }
-.form-footer{
+.form-footer-jitsi{
```
Comment (Rupeshiya) [inline src/user/integrations/jitsi.scss:70]: Remove unnecessary CSS classes that are not used, please.

**Comment by Rupeshiya** [comment]
> Final UI.
> finalll

Looks fine now.

> Final UI.
> finalll

Looks good!!

File: src/user/integrations/jitsi.scss (hunk 3 above)
Comment (nisarg1499) [reply src/user/integrations/jitsi.scss]: Removed unnecesarry css classes from jitsi.scss.

**Pull request merged**